launcher/launcher/main.go

```diff
@@ -171,18 +193,26 @@ func startLauncher(ctx context.Context, launchSpec spec.LaunchSpec, serialConsol
 	}
 	gceAk.Close()
 
-	token, err := registryauth.RetrieveAuthToken(ctx, mdsClient)
+	token, err := registryauth.RetrieveAuthToken(context.Background(), mdsClient)
+	if err != nil {
+		logger.Info(fmt.Sprintf("failed to retrieve auth token: %v, using empty auth for image pulling\n", err))
+	}
+
+	uptime, err := getUptime()
```

Comment:
why need to get the system up time here? I thought we want the seconds since `main()` started?

Reply:
main() only takes a few seconds to run (at least in my test machines), I thought the time from VM start would be more useful but if not it can be switched back. wdyt?

Reply:
I think both are useful to track how long boot takes and how long the launch itself takes